src/content/app/tools/blast/components/blast-job-submit/BlastJobSubmit.tsx

```diff
@@ -30,6 +30,8 @@ import { getSelectedSpeciesIds } from 'src/content/app/tools/blast/state/blast-f
 
 import { toFasta } from 'src/shared/helpers/formatters/fastaFormatter';
 
+import { PrimaryButton } from 'src/shared/components/button/Button';
```

Comment:
Group this import with `LoadingButton` above?

Reply:
I am confused. This file doesn't import `LoadingButton`.

Reply:
I made the comment before the component was removed. This no longer applies then 😁